app/services/utility-service.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const ServerConfig = require('../configs/server-config');
+
+class UtilityService {
+    constructor(playerContainer, notificationService) {
+        this.playerContainer = playerContainer;
+        this.notificationService = notificationService;
+
+        this.currentFPS = ServerConfig.STARTING_FPS;
+    }
+
+    changeSpeed(playerId, speedOption) {
```

Comment:
Just for fun :P

Reply:
lagger ikke pga dette?

Reply:
ikke som jeg opplevde, men har ikke testet med mange spillere. enkelt å fjerne uansett :P